### PR TITLE
[V3 Audio] Embeds colour respecting its set value

### PR DESCRIPTION
**Pull request by Brramble**
### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### Description of the changes

I have gone through the cog and removed all set colours from what it used to be and replaced with the embed request so it should now run uniform to the rest of the bot.

### CODE REVIEW

File: redbot/cogs/audio/audio.py [hunk 1]
```diff
@@ -130,9 +129,12 @@ def __init__(self, bot):
...
                 notify_channel = self.bot.get_channel(notify_channel)
                 embed = discord.Embed(
-                    colour=notify_channel.guild.me.top_role.colour, title="Queue ended."
+                    title="Queue ended."
                 )
-                await notify_channel.send(embed=embed)
+                if await ctx.embed_requested():
+                    await notify_channel.send(embed=embed)
+                else:
+                    await notify_channel.send(embed=embed)
```
Comment (Tobotimus) [inline redbot/cogs/audio/audio.py]: This isn't doing anything. The condition statement is meaningless - the bot will send the exact same thing no matter the condition.

**Review by Tobotimus — CHANGES_REQUESTED** [review]
I appreciate what you're trying to do here. I think the function you're looking for is `await ctx.embed_colour()`. Checking `if ctx.embed_requested()` is just checking whether embeds are enabled, for which you're sending an embed if they're disabled anyway. Best remove all of those statements.

**Comment by Brramble** [comment]
Thank you for the help I will rework it.

**Comment by Brramble** [comment]
Ive also added a [p]thumbnail command which will display the songs thumbnail in an embed. Furthermore with thumbnails in the [p]queue and [p]now commands. 

Me being stupid I pushed it all at once so allow me to go back to remove them for this PR as this is just for embeds

**Pull request closed without merging**